.travis_dev.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js: "8"
+script:
+  - yarn lint
+  - yarn test
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && ([ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "staging" ]); then yarn run build; fi
```

Comment:
Should add a branch in this conditional, probably the `ep-upgrade` branch

Reply:
@wrandall22 ep-upgrade branch was added.

Reply:
Updated with -
[ "$TRAVIS_PULL_REQUEST" = "false" ] && ([ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "ep-upgrade" ])